### PR TITLE
*: update prow OWNERS_ALIASES file

### PR DESCRIPTION
**Pull request by wuhuizuo**
### What problem does this PR solve?

Issue Number: ref #46911

Problem Summary:

### What changed and how does it work?

Reverts pingcap/tidb#54174

Although the code owners of a directory do not conflict with the community member mechanism of the repository, the exit mechanism of the code owners of the directory still needs to be determined(ref: https://internals.tidb.io/t/topic/877
).

Only the repository "/OWNERS" file will be synchronized with the member list in [pingcap/community](https://github.com/pingcap/community), in which members can participate in the review of the entire repository. However, for subdirectories configured with the "OWNERS" file, the code owners of the directory are needed to determine whether they can be merged.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [x] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -30,7 +30,7 @@ aliases:
   sig-approvers-executor-import: # approvers for importer part of executor pkg.
     - D3Hunter
     - gmhdbjd
-    - lance6716
+    - lance6716
```
Comment (JmPotato) [inline OWNERS_ALIASES:33]: Typo?

Comment (wuhuizuo) [inline OWNERS_ALIASES:33]:
```suggestion
    - lance6716
```

Comment (wuhuizuo) [reply OWNERS_ALIASES]: Oh, it's a history blank space mistake. Fixed.

**Review by ywqzzy — APPROVED** [review]

**Pull request merged**